mne/defaults.py

```diff
@@ -15,19 +15,19 @@
     units=dict(mag='fT', grad='fT/cm', eeg='µV', eog='µV', ecg='µV', emg='µV',
                misc='AU', seeg='mV', dipole='nAm', gof='GOF', bio='µV',
                ecog='µV', hbo='µM', hbr='µM', ref_meg='fT', fnirs_raw='V',
-               fnirs_od='V', csd='V/m²'),
+               fnirs_od='V', csd='mV/m²'),
     # scalings for the units
     scalings=dict(mag=1e15, grad=1e13, eeg=1e6, eog=1e6, emg=1e6, ecg=1e6,
                   misc=1.0, seeg=1e3, dipole=1e9, gof=1.0, bio=1e6, ecog=1e6,
                   hbo=1e6, hbr=1e6, ref_meg=1e15, fnirs_raw=1.0, fnirs_od=1.0,
-                  csd=1e5),
+                  csd=1e3),
     # rough guess for a good plot
     scalings_plot_raw=dict(mag=1e-12, grad=4e-11, eeg=20e-6, eog=150e-6,
                            ecg=5e-4, emg=1e-3, ref_meg=1e-12, misc='auto',
                            stim=1, resp=1, chpi=1e-4, exci=1, ias=1, syst=1,
                            seeg=1e-4, bio=1e-6, ecog=1e-4, hbo=10e-6,
                            hbr=10e-6, whitened=10., fnirs_raw=2e-2,
-                           fnirs_od=2e-2, csd=20e-4),
+                           fnirs_od=2e-2, csd=200e-4),
```

Comment:
This scale factor, on the other hand, just says how to scale the SI-unit data to make it generally look nice in `raw.plot`, so it does not need to maintain the scale-factor-to-SI relationship.